src/main/java/mods/natura/common/NContent.java

```diff
@@ -1110,9 +1110,6 @@ public void addRecipes() {
         // Clouds
         GameRegistry.addRecipe(stackSingleCharcoal, "ccc", "ccc", "ccc", 'c', new ItemStack(cloud, 1, 2));
         GameRegistry.addRecipe(new ItemStack(plantItem, 1, 4), "cc", "cc", 'c', new ItemStack(cloud, 1, 3));
-        GameRegistry.addRecipe(
-                new ShapedOreRecipe(new ItemStack(Items.gunpowder, 1, 0), "cc", "cc", 'c', "dustSulfur"));
```

Comment:
Please don't remove the recipe. Just put it in a `if (!Loader.isModLoaded("dreamcraft")) { }`  block instead.